### PR TITLE
Delete createPR.md

### PR DESCRIPTION
**Pull request by codercool3r**
Modified the description for the pull request guide and the first-timers guide

(Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)

### Does this close any currently open issues?

Addresses # (issue)  - use this if your pull request solves only a part of the issue

### Any relevant logs, error output, etc?
(If it’s long, please paste to https://ghostbin.com/ and insert the link here.)

### Any other comments?
....

### CODE REVIEW

**Comment by codercool3r** [comment]
> Modified the description for the pull request guide and the first-timers guide
> 
> (Thanks for sending a pull request! Please make sure you click the link above to view the contribution guidelines, then fill out the blanks below.)
> 
> ### What does this implement/fix? Explain your changes.
> ### Does this close any currently open issues?
> Addresses # (issue) - use this if your pull request solves only a part of the issue
> 
> ### How did you test your changes?
> ### Any relevant logs, error output, etc?
> (If it’s long, please paste to https://ghostbin.com/ and insert the link here.)
> 
> ### Any other comments?
> ....

I have done this under GSSOC'22

**Comment by siv2r** [comment]
Hey, thanks for the PR. There is already a `docs/newContributors.md` which does this job.

You can simply delete the `docs/createPR.md` file. That is a good change to do.

**Comment by codercool3r** [comment]
> Hey, thanks for the PR. There is already a `docs/newContributors.md` which does this job.
> 
> You can simply delete the `docs/createPR.md` file. That is a good change to do.

Oh, sorry about that, since the files had different content, I figured its better to add it.
I will remove the file.

**Comment by siv2r** [comment]
You don't need to close the issue. You can just update your current git branch and force push. This will update your PR on GitHub.

File: README.md [hunk 1]
```diff
@@ -49,10 +49,10 @@ After a doctor completes this process, this platform provides two main features.
...
 
 ## Documentation
-[First timers Guide](./docs/newContributors.md): **New Contributors** follow this to create **your first Pull Request**.  
-[Contributing Guide](./docs/CONTRIBUTING.md): Please follow the rules mention here while contributing to this project.  
+[First timers Guide](./docs/newContributors.md): **New Contributors** follow this to create **your first Pull Request**. 
+[Contributing Guide](./docs/CONTRIBUTING.md): Please follow the rules mention here while contributing to this project.
 [Build documentation](./docs/build.md): Follow this to setup the project in your local machine.  
-[Scoring Functions](https://github.com/siv2r/kidney-exchange/projects/1): Parameters used for find the match score.  
+[Scoring Functions](https://github.com/siv2r/kidney-exchange/projects/1): Parameters used for find the match score.
 [FAQs](./docs/FAQs.md): Frequently asked questions
```
Comment (siv2r) [inline README.md:56]: These changes mess up the alignment when the readme file is rendered. Can you undo all the changes made in this file?

**Review by siv2r — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (codercool3r) [reply README.md:56]: Really sorry, I am kinda new to this, didn't mean to mess it up.

**Comment by codercool3r** [comment]
Can you check now

File: README.md (hunk 1 above)
Comment (siv2r) [reply README.md]: No worries

**Comment by siv2r** [comment]
Changes looks good! Can you squash your commit?

**Comment by codercool3r** [comment]
> Changes looks good! Can you squash your commits (from 4 commits to one )?
I have squashed, but I am not able to check if it actually happened. Can you please confirm

**Comment by siv2r** [comment]
No, the squash is not reflected. You need to force push (`git push -f origin branch-name`) after squashing your commits.

**Comment by codercool3r** [comment]
> No, the squash is not reflected. You need to force push (`git push -f origin branch-name`) after squashing your commits.

Done, thank you soo much for being patient with my questions, even if this was a very minor pull request. :)
Also, Sorry for kinda messing things up.

**Comment by siv2r** [comment]
> Also, Sorry for kinda messing things up.

No worries, that is part of the learning process :)

Also, the changes in the README are included again 😂. Can you remove those changes?

**Comment by codercool3r** [comment]
Actually, since this is just a minor issue, and also I am not able to figure it out. I am closing this pull request, sorry for all the unnecessary chaos.

**Pull request closed without merging**